extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPlugin.scala

```diff
@@ -109,8 +109,6 @@ object SparkRangerAdminPlugin extends RangerBasePlugin("spark", "sparkSql")
       } else if (result.getMaskTypeDef != null) {
         result.getMaskTypeDef.getName match {
           case "MASK" => regexp_replace(col)
-          case "MASK_SHOW_FIRST_4" if isSparkV31OrGreater =>
-            regexp_replace(col, hasLen = true)
```

Comment:
seems we should remove the next branch instead of this one

Reply:
Sorry, it's my mistake. Fixed.